specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2021-03-01-preview/examples/ConfigurationStoresCreate.json

```diff
@@ -0,0 +1,75 @@
+{
+  "parameters": {
+    "subscriptionId": "c80fb759-c965-4c6a-9110-9b2b2d038882",
+    "resourceGroupName": "myResourceGroup",
+    "configStoreName": "contoso",
+    "api-version": "2021-03-01-preview",
+    "configStoreCreationParameters": {
+      "location": "westus",
+      "sku": {
+        "name": "Standard"
+      },
+      "tags": {
+        "myTag": "myTagValue"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "type": "Microsoft.AppConfiguration/configurationStores",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "creationDate": "2018-04-24T16:30:55+00:00",
+          "endpoint": "https://contoso.azconfig.io",
+          "encryption": {
+            "keyVaultProperties": {
+              "keyIdentifier": null,
+              "identityClientId": null
+            }
+          },
+          "disableLocalAuth": false,
```

Comment:
>disableLocalAuth [](start = 11, length = 16)

since this is not a read-only property you should add examples where this is also specified in the request body for put and patch

Reply:
I will add examples and update the PR.

Reply:
@varunkumta Looks this could be a validation rule candidate in our swagger validation pipeline.  How about adding a new rule to check swagger property coverage? make sure all non readonly property must have a related example.